docs/quickstart.rst

```diff
@@ -0,0 +1,77 @@
+.. highlight:: shell
+
+==========
+Quickstart
+==========
+
+
+Importing dask-image
+--------------------
+Import dask image is with an underscore, like this example:
+
+.. code-block:: python
+
+    import dask_image.imread
+    import dask_image.ndfilters
+
+
+Dask Examples
+-------------
+We highly recommend checking out the dask-image-quickstart.ipynb notebook 
+(and any other dask-image example notebooks) at the dask-examples repository:
+
+https://github.com/dask/dask-examples
+
+You can find the dask-image quickstart notebook in the `applications` folder
+of this repository. All the example notebooks are available to launch with
+mybinder and test out interactively.
+
+
+An Even Quicker Start
+---------------------
+
+You can read files stored on disk into a dask array
+by passing the filename, or regex matching multiple filenames
+into `imread()`.
+
+.. code-block:: python
+
+    filename_pattern = 'path/to/image-*.png'
+    images = dask_image.imread.imread(filename_pattern)
+
+If your images are parts of a much larger image, 
+dask can stack, concatenate or block chunks together:
+http://docs.dask.org/en/latest/array-stack.html
+
+
+Calling dask-image functions is also easy.
+.. code-block:: python
+
+    import dask_image.ndfilters
+    blurred_image = dask_image.ndfilters.gaussian_filter(images, sigma=10)
+
+
+If you have a function that accepts input numpy arrays,
+it is also likely to accept dask arrays without any modification. 
```

Comment:
I'm not sure about this statement. There are a few cases this might run into issues (e.g. calls to `asarray` or some NumPy functions). Plus it's probably more general than what we'd like to discuss in dask-image (i.e. maybe a statement like this belongs in Dask's docs). Would it make sense to link out to Dask Array documentation for more general array operations or is there something in particular we'd like to say about image data here?

Reply:
Yes, that would be the careful and sensible thing to do here. Changed & link added.